management/virtualmachineimage/client.go

```diff
@@ -19,10 +20,33 @@ func NewClient(client management.Client) Client {
 	return Client{client}
 }
 
-func (c Client) ListVirtualMachineImages() (ListVirtualMachineImagesResponse, error) {
+//ListVirtualMachineImages lists the available VM images, filtered by the optional parameters.
+//See https://msdn.microsoft.com/en-us/library/azure/dn499770.aspx
+func (c Client) ListVirtualMachineImages(parameters *ListParameters) (ListVirtualMachineImagesResponse, error) {
```

Comment:
Ugly. Writing nil for no optional parameters is much nicer. Want me to fix anyway?